templates/taxi/car_detail.html

```diff
@@ -1,12 +1,16 @@
 {% extends "base.html" %}
 
 {% block content %}
+  <h1>Car Details</h1>
+  <p>Id {{ car.id }}</p>
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
+  <h2>Drivers</h2>
   <hr>
   <ul>
   {% for driver in car.drivers.all %}
     <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
   {% endfor %}
   </ul>
+  <a href="{% url 'taxi:car-update' pk=car.id %}">UPDATE CAR</a>
+  <a href="{% url 'taxi:car-delete' pk=car.id %}">DELETE CAR</a>
```

Comment:
According to the checklist, you should add a 'Cancel' button apart from the 'Delete' one. The 'Cancel' button should lead to the previous page the user was on.